src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,58 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+        Background:
+            Given a school exists
+                  | schoolId  | nameOfInstitution      | gradeLevels       | educationOrganizationCategories |
+                  | 255901001 | Grand Bend High School | [ "Tenth grade" ] | [ "School" ]                    |
+              And there is a valid AcademicWeek
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | Week One       | 2024-05-15 | 2024-05-15 | 2                      |
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by valid date
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate={beginDate}"
+             Then it should respond with 200
+              And response contains "weekIdentifier: "string" "
+
+        @ignore
+        Scenario: Ensure clients can't GET information when querying by invalid date
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=024-04-09"
+             Then it should respond with 400
+              And response body
```

Comment:
This should be applied to the other scenarios as well, we need to be consistent in the used methods; instead of having 4 different validations in the response body, we could reuse the existing one
`              And the response body is`
Also use the entire json in the response body as well

Reply:
Complete